app/src/shared/hooks/useRequireNativeTokenBalance.js

```diff
@@ -1,51 +0,0 @@
-import { useMemo, useCallback, useState, useEffect } from 'react'
-
-import { getEthBalance } from '$mp/utils/web3'
-import usePending from '$shared/hooks/usePending'
-import useIsMounted from '$shared/hooks/useIsMounted'
-
-const useRequireNativeTokenBalance = (minimumBalance = 0) => {
-    const { isPending, wrap } = usePending('balance.CHECK')
-    const isMounted = useIsMounted()
-    const [hasBalance, setHasBalance] = useState(undefined)
-
-    const checkBalance = useCallback((requiredBalance = minimumBalance) => (
-        wrap(async () => {
-            try {
-                const balance = await getEthBalance()
```

Comment:
@tumppi this needed an additional argument – the address. Fixed in the refactored version.